shell/platform/android/test/io/flutter/embedding/android/FlutterActivityTest.java

```diff
@@ -94,6 +96,48 @@ public void flutterViewHasId() {
     assertTrue(activity.findViewById(FlutterActivity.FLUTTER_VIEW_ID) instanceof FlutterView);
   }
 
+  @Test
+  @Config(sdk = API_LEVELS.API_34)
```

Comment:
please make this minSdk not sdk

Reply:
Done in both places, and in `FlutterFragmentActivityTest.java`